packages/web/components/Player/PlayerTile.tsx

```diff
@@ -91,21 +94,27 @@ export const PlayerTile: React.FC<Props> = ({
               borderRadius={10}
               fontFamily="body"
               fontWeight={400}
+              textShadow="0 0 8px var(--chakra-colors-blackAlpha-400)" // v. light shadow makes the text readable if the logo/avatar is white
             >
               {getPlayerName(player)}
             </Heading>
           </Flex>
         </MetaTileHeader>
         <MetaTileBody pos="relative" height="full">
-          <Flex direction="column" mb="auto">
+          {/**
+           * The mb="auto" pushes the last block (DAO memberships/Contact) down to the bottom of the tile
+           */}
+          <Flex direction="column" gap={2} mb="auto">
             {displayDescription && (
-              <VStack spacing={2} align="stretch">
+              <VStack spacing={1} align="stretch">
                 <Text textStyle="caption">About</Text>
-                <Text fontSize="sm">{displayDescription}</Text>
+                <Text fontSize="sm" noOfLines={4}>
+                  {displayDescription}
```

Comment:
Personally, I'd render the Markdown here so that there aren't formatting characters in the text, and so that if there are paragraphs they'll be displayed.

It does open the possibility though of someone taking up too much space by using headers, so, it's up to your discretion.

```suggestion
                  <Markdown>{description}</Markdown>
```

Reply:
I'm going to leave this as-is for now as I'm not sure what kind of text sanitisation/conversion is going on in the background.

I did a quick test and it looked like tags were getting converted (< to &lt;, etc)

Opening it up to Markdown would let people add images and more (potentially breaking the layout?) and I think there should be a discussion around that first.